extensions/interrupts/README.md

```diff
@@ -45,7 +45,7 @@ The following table specifies which flag is associated with which bit in the `FL
 |:-------------------------|:-------------|:-----------------|:-------------------------------------------------------------------------------------------------------------------|
 | System Call              | Synchronous  | 0                | Used by programs to ask the operating system to do something.                                                      |
 | Timer                    | Asynchronous | 1                | Occurs when a timer causes an interrupt.                                                                           |
-| Illegal Instruction      | Synchronous  | 2                | Occurs when execution of an undefined, reserved, or illegal instruction is attempted.                              |
+| Illegal Instruction      | Synchronous  | 2                | Occurs when execution of a reserved, or illegal instruction is attempted.                                          |
 | Memory Alignment Error   | Synchronous  | 3                | Occurs when an attempt to read or write to memory at an unaligned address occurs and is not supported.             |
 | General Protection Fault | Synchronous  | 4                | Occurs when the CPU needs the operating system to handle an unexpected event for stability or consistency reasons. |
```

Comment:
Do we care about specifying/explaining the difference between an _interrupt_ (something needs to happen, but the program is fine) and an _exception/fault_ (the program has done something wrong and needs to be corrected or terminated)?

What about giving these convenient (but suggestive) abbreviations for reference elsewhere, such as `#UD`, `#AC`, and `#GP`? Syntax ala x86, but just a placeholder.

Reply:
I think differentiating between an interrupt and exception is probably a good idea. I'm neutral on creating abbreviations.

Reply:
I agree with Ender in both points.